localize/comments/15/NuGet.CommandLine.XPlat.dll.lci

```diff
@@ -404,11 +431,11 @@
     <Disp Icon="Ver" Disp="true" LocTbl="false" Path=" \ ;Version \ 8 \ 0" />
     <Item ItemId=";Comments" ItemType="0" PsrId="211" InstFlg="true" Leaf="true">
       <Str Cat="Text">
-        <Val><![CDATA[NuGet wrapper for dotnet.exe.]]></Val>
+        <Val><![CDATA[NuGet executable wrapper for the dotnet CLI nuget functionality.]]></Val>
```

Comment:
Should second `nuget` be `NuGet` ?

Reply:
That's the description from project file. I image it refers to `dotnet nuget` command name.

https://github.com/NuGet/NuGet.Client/blob/343184861e337a695fffb0cf05be4aee23ae6823/src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj#L6